docs: add traces support document (#5688)
jina-ai/jina: docs/concepts/jcloud/yaml-spec.md

    - `user`
    - `jina`-version
```

### Monitoring

> To enable [tracing support](https://docs.jina.ai/cloud-nativeness/opentelemetry/) in Flows, you can pass `enable: true` argument in the Flow YAML. (By default, Jcloud will disable the tracing support)

Reviewer: ```suggestion
To enable [tracing support](https://docs.jina.ai/cloud-nativeness/opentelemetry/) in Flows, you can pass `enable: true` argument in the Flow YAML. (Tracing support is not enabled by default in JCloud)
```

```{code-block} yaml
---
emphasize-lines: 2-5
---
jtype: Flow
jcloud:
  monitor:
    traces:
      enable: true
executors:
  - name: executor1
    uses: jinaai+docker://<username>/Executor1
```

> You can pass the `enable: true` argument to `gateway`, so as to only enable the tracing support in gateway.

Reviewer: ```suggestion
You can pass the `enable: true` argument to `gateway`, so as to only enable the tracing support in gateway:
```
Use `:` before code samples throughout
